web/src/pages/LogoutPage/LogoutPage.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+import { Redirect } from "react-router-dom";
+import UserService from "../../services/user";
+
+class LogoutPage extends Component {
```

Comment:
Does it have to be a page? I mean, it works. But it's really necessary?

Reply:
It's necessary in the sense that it pushes a page to history before redirecting to /. Otherwise if you're already in /, it doesn't do anything. Maybe there's a programmatic way of doing it, but I thought maybe just redirecting to a component that does all the logout part would be simpler